packages/core/contracts/financial-templates/long-short-pair/LongShortPair.sol

```diff
@@ -280,7 +280,7 @@ contract LongShortPair is Testable, Lockable {
      * @param sponsor address of the sponsor to query.
      * @return [uint256, uint256]. First is long tokens held by sponsor and second is short tokens held by sponsor.
      */
-    function getPositionTokens(address sponsor) public view returns (uint256, uint256) {
+    function getPositionTokens(address sponsor) public view nonReentrantView() returns (uint256, uint256) {
```

Comment:
Is this the only functional change requested?

Reply:
yupo!